di_website/common/templatetags/string_utils.py

```diff
@@ -71,14 +72,16 @@
 
 @register.filter(name='wysiwyg_tags')
 def wysiwyg_tags(text):
-    return mark_safe(bleach.clean(
+    clean = bleach.clean(
         text,
         tags=WYSIWYG_LIST,
         attributes=ATTRS,
         styles=STYLES,
         strip=True,
         strip_comments=True
-    ).replace('<p> </p>', ''))
+    )
+
+    return mark_safe(re.sub(r'<p>\s*</p>', '', clean))
```

Comment:
@akmiller01 not great at regular expressions, so not sure if this will do for all paragraph tags with any number of spaces... seems to be working, but... hmm